### PR TITLE
document data-model create/restore

### LINKED ISSUE #31: enhancement: document how to create a new datamodel
Looking through the docs I noticed there's no documentation on how to create a new datamodel.

### PR DESCRIPTION
**Pull request by tudorbarascu**
Fixes #31

### CODE REVIEW

File: en/installation-guide/database-initialization.rst [hunk 1]
```diff
@@ -84,3 +91,42 @@ You can do this graphically or by simply opening a pgAdminIII :ref:`SQL-query` w
...
+* Setup the ``pg_service.conf`` file and add the ``pg_qgep`` service
+  as in the :ref:`pg_service-linux` section.
+
+* Create the database::
+
+   psql -U postgres -c 'CREATE DATABASE qgep;'
+
+* Run the generation script::
+
+   scripts/dbsetup.sh
```
Comment (m-kuhn) [inline en/installation-guide/database-initialization.rst]: should that be prefixed with a `./` to make it easier?

File: en/installation-guide/database-initialization.rst [hunk 2]
```diff
@@ -84,3 +91,42 @@ You can do this graphically or by simply opening a pgAdminIII :ref:`SQL-query` w
...
+
+If you want to use a different SRID you need to use the ``-s`` option.
+For instance, run ``scripts/dbsetup.sh -s 3857`` for the **3857** SRID.
```
Comment (m-kuhn) [inline en/installation-guide/database-initialization.rst]: can we change the example to 2056 or add it as an option with description?

Comment (tudorbarascu) [inline en/installation-guide/database-initialization.rst]: Done. Thanks.

**Comment by tudorbarascu** [comment]
ping @m-kuhn Should be good now

**Comment by m-kuhn** [comment]
Thank you @tudorbarascu

**Pull request merged**